cssfiles/footer.css

```diff
@@ -0,0 +1,29 @@
+.footer-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--primary-color);
+    color: var(--primary-text-color);
+    padding: 1rem;
```

Comment:
Can use globally defined variables for spacing properties like margin, padding, and gap to maintain consistency throughout the website

Reply:
ok man will consider that.